src/app/portlets/content-types/edit/content-types-edit.component.ts

```diff
@@ -297,15 +297,15 @@ export class ContentTypesEditComponent implements OnInit, OnDestroy {
     }
 
     private handleHttpError(err: ResponseView) {
-        this.dotHttpErrorManagerService.handle(err).subscribe((_handled: DotHttpErrorHandled) => {
+        this.dotHttpErrorManagerService.handle(err).pipe(take(1)).subscribe((_handled: DotHttpErrorHandled) => {
             this.dotRouterService.gotoPortlet('/content-types-angular');
         });
     }
 
     private updateContentType(value: any): void {
         const data = Object.assign({}, value, { id: this.data.id });
 
-        this.crudService.putData(`v1/contenttype/id/${this.data.id}`, data).subscribe(
+        this.crudService.putData(`v1/contenttype/id/${this.data.id}`, data).pipe(take(1)).subscribe(
```

Comment:
maybe into the service

Reply:
We can do this right now... we need to create an issue for it.